src/Map.jsx

```diff
@@ -75,6 +76,30 @@ class Map extends Component {
     );
   }
 
+  getStopInfo(route, stopCoordinates) {
+    let stops = this.state.selectedStops;
+    if (stops.length === 0) {
+      stops.push(stopCoordinates);
+      const station = this.findStationID(route, stopCoordinates);
+      console.log(`First Stop ID: ${station.sid}`);
+    } else if (stops.length === 1 && stops[0] !== stopCoordinates) {
+      stops.push(stopCoordinates);
+      const station = this.findStationID(route, stopCoordinates);
+      console.log(`Second Stop ID: ${station.sid}`);
+    } else {
+      stops = [];
+      stops.push(stopCoordinates);
+      const station = this.findStationID(route, stopCoordinates);
+      console.log(`First Stop ID: ${station.sid}`);
+    }
+    this.setState({ selectedStops: stops });
+  }
+
+  findStationID(route, stopCoord) {
+    console.log(this);
+    const index = route.stops.findIndex(c => c.lon === stopCoord[0] && c.lat === stopCoord[1]);
```

Comment:
@EddyIonescu weird naming again b/c of 100 character limit (though your suggestion on the other line by breaking up the ternary operator worked)
Using findIndex which returns first index when condition is true